x-pack/plugins/endpoint/common/types.ts

```diff
@@ -115,6 +131,18 @@ export interface EndpointMetadata {
   };
 }
 
+export interface ESTotal {
+  value: number;
+  relation: string;
+}
+
+export interface AlertDataWrapper {
+  _id: string;
+  _source: AlertData;
+}
+
+export type AlertHits = AlertDataWrapper[];
```

Comment:
Do we need this? Wouldn't `SearchResponse<AlertData>` serve the same purpose?

Reply:
`SearchResponse` contains 2 levels of hits. This only contains the inner layer, and it's needed for mapping results which contain an `_id` and a `_source`. There doesn't appear to be an existing type that covers that.